src/main/groovy/nebula/plugin/publishing/ivy/IvyPublishPlugin.groovy

```diff
@@ -29,5 +31,8 @@ class IvyPublishPlugin implements Plugin<Project> {
         project.plugins.apply IvyManifestPlugin
         project.plugins.apply IvyRemovePlatformDependenciesPlugin
         project.plugins.apply IvyRemoveInvalidDependenciesPlugin
+        project.plugins.apply SpringBootJarPlugin
```

Comment:
Nice! Users do not need to change the plugins they are applying, but if they use the Shadow plugin of Spring Boot plugin, then these modifications are automatically applied 👍

Reply:
yeah! that's the plan! since folks using nebula have the opinion of using `components.java`, we should apply these new opinions, too!